site/en/blog/extension-manifest-converter/index.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: 'layouts/blog-post.njk'
+title: Extension Manifest Converter
+description: >
+  Open source tool to convert extensions to Manifest V3. You'll still need to manually update any
+  service worker code that relies on a DOM, in addition to updating chrome.scripts.
+subhead: >
+  Easily convert an entire directory, extension zip file, or manifest.json file.
+date: 2021-04-28
+updated: 2021-04-28
+authors:
+  - solomonkinard
+  - dotproto
+tags:
+  - extensions
+hero: image/WlD8wC6g8khYWPJUsQceQkhXSlv1/GVDuhpIli5DeYmbiPlqG.jpg
```

Comment:
While a good picture - what's the correlation between planing and MV2->3 conversion? ; )

Reply:
It's a pretty loose association. The vague idea is that like a plane this is specialized tool for a specific job. It won't do everything for you, but it is useful for the niche it fills. Nothing beyond that besides its visual appeal.

I'm open to other ideas. I also have a couple other contenders that might be worth discussing offline.